src/nvme/types.h

```diff
@@ -4658,6 +4658,20 @@ struct nvme_supported_cap_config_list_log {
 	struct nvme_capacity_config_desc cap_config_desc[];
 };
 
+/**
+ * struct nvme_lockdown_log - Command and Feature Lockdown Log
+ * @csss:	Contents Selected and Scope Selected
+ * @rsvd1:	Reserved
+ * @length:	Length of Command and Feature Identifier List field
+ * @cfil:	Command and Feature Identifier List
+ */
+struct nvme_lockdown_log {
+	__u8	csss;
+	__u8	rsvd1[2];
+	__u8	length;
```

Comment:
`lngth` is used as field name in the spec, I  think we should also use it here, just to be consistent.